src/test/java/hudson/plugins/analysis/collector/workflow/WorkflowCompatibilityTest.java

```diff
@@ -0,0 +1,86 @@
+package hudson.plugins.analysis.collector.workflow;
+
+import static org.junit.Assert.*;
```

Comment:
Yes, but it is common to make an exception for `Assert` and perhaps `Assume`, `Matchers`, etc.